### PR TITLE
Add the verifier contract address to docs for v0.10.0

### PR DESCRIPTION
**Pull request by nategraf**
Add the verifier contract address to docs for v0.10.0

### CODE REVIEW

**Review by capossele — APPROVED** [review]

**Comment by pdg744** [comment]
Is `v0.10.0` a typo in the PR?

**Comment by pdg744** [comment]
Ohh, that's based on the numbering on `risc0-ethereum`. 
Do we need/want to do anything to clarify that the versioning here is not related to the zkvm versioning?

File: website/api/blockchain-integration/contracts/verifier.md [hunk 1]
```diff
@@ -54,20 +54,24 @@ We are working on process to provide opt-in upgradeability for the verifier cont
...
 
-| Contract                      | Network   | Address                                                        |
-| ----------------------------- | --------- | -------------------------------------------------------------- |
-| [RiscZeroGroth16Verifier.sol] | [Sepolia] | [0xe57255C10291771A2E14f7eb257555AE82D78302][sepolia-verifier] |
+| Contract                      | Version                      | Network   | Address                                                                |
+| ----------------------------- | ---------------------------- | --------- | ---------------------------------------------------------------------- |
+| [RiscZeroGroth16Verifier.sol] | [v0.9.0][contracts-v0.9.0]   | [Sepolia] | [0xe57255C10291771A2E14f7eb257555AE82D78302][sepolia-verifier-v0.9.0]  |
+| [RiscZeroGroth16Verifier.sol] | [v0.10.0][contracts-v0.10.0] | [Sepolia] | [0x4e9559f1fdf82dd1babdd42b4550ffdb1a1c59de][sepolia-verifier-v0.10.0] |
 
 [RiscZeroGroth16Verifier.sol]: https://github.com/risc0/risc0-ethereum/blob/release-0.8/contracts/src/groth16/RiscZeroGroth16Verifier.sol
```
Comment (pdg744) [inline website/api/blockchain-integration/contracts/verifier.md:62]: Should we update this to release-0.10?

File: website/api/blockchain-integration/contracts/verifier.md [hunk 2]
```diff
@@ -54,20 +54,24 @@ We are working on process to provide opt-in upgradeability for the verifier cont
 RISC Zero provides an already deployed verifier contract in the Sepolia network for your convenience.
 You can choose to use this contract or deploy your own.
 
-| Contract                      | Network   | Address                                                        |
-| ----------------------------- | --------- | -------------------------------------------------------------- |
-| [RiscZeroGroth16Verifier.sol] | [Sepolia] | [0xe57255C10291771A2E14f7eb257555AE82D78302][sepolia-verifier] |
+| Contract                      | Version                      | Network   | Address                                                                |
+| ----------------------------- | ---------------------------- | --------- | ---------------------------------------------------------------------- |
+| [RiscZeroGroth16Verifier.sol] | [v0.9.0][contracts-v0.9.0]   | [Sepolia] | [0xe57255C10291771A2E14f7eb257555AE82D78302][sepolia-verifier-v0.9.0]  |
```
Comment (pdg744) [inline website/api/blockchain-integration/contracts/verifier.md:59]: Currently clicking on either of these contract links will land you on the v0.8 version of the contract.

File: website/api/blockchain-integration/contracts/verifier.md [hunk 3]
```diff
@@ -54,20 +54,24 @@ We are working on process to provide opt-in upgradeability for the verifier cont
...
-[sepolia-verifier]: https://sepolia.etherscan.io/address/0xe57255C10291771A2E14f7eb257555AE82D78302#code
+[sepolia-verifier-v0.9.0]: https://sepolia.etherscan.io/address/0xe57255C10291771A2E14f7eb257555AE82D78302#code
+[sepolia-verifier-v0.10.0]: https://sepolia.etherscan.io/address/0x4e9559f1fdf82dd1babdd42b4550ffdb1a1c59de#code
 [term-journal]: /terminology#journal
 [term-receipt]: /terminology#receipt
 [term-verify]: /terminology#verify
 [term-image-id]: /terminology#image-id
 [term-zkvm-program]: /terminology#zkvm-program
 [foundry-template]: https://github.com/risc0/bonsai-foundry-template
+[contracts-v0.9.0]: https://github.com/risc0/risc0-ethereum/releases/tag/v0.9.0
```
Comment (pdg744) [inline website/api/blockchain-integration/contracts/verifier.md:76]: Are we better off pointing to the release notes or to the [contract from that release](https://github.com/risc0/risc0-ethereum/blob/v0.9.0/contracts/src/IRiscZeroVerifier.sol)?

File: website/api/blockchain-integration/contracts/verifier.md [hunk 4]
```diff
@@ -54,20 +54,24 @@ We are working on process to provide opt-in upgradeability for the verifier cont
 RISC Zero provides an already deployed verifier contract in the Sepolia network for your convenience.
 You can choose to use this contract or deploy your own.
 
-| Contract                      | Network   | Address                                                        |
-| ----------------------------- | --------- | -------------------------------------------------------------- |
-| [RiscZeroGroth16Verifier.sol] | [Sepolia] | [0xe57255C10291771A2E14f7eb257555AE82D78302][sepolia-verifier] |
+| Contract                      | Version                      | Network   | Address                                                                |
```
Comment (pdg744) [inline website/api/blockchain-integration/contracts/verifier.md:57]: Do we want a column here indicating the control root and/or list of allowed programs? 
Or perhaps we want a separate table for tracking versioning for the control root?

Comment (pdg744) [reply website/api/blockchain-integration/contracts/verifier.md:57]: I do like the simplicity of this table, so I'm hesitant to complicate it with the very technical implementation detail of the control root. 

On the other hand, as a user, I'd like to have a way to confirm that the control root being used corresponds to the set of `zkr` programs I trust.

File: website/api/blockchain-integration/contracts/verifier.md (hunk 2 above)
Comment (nategraf) [reply website/api/blockchain-integration/contracts/verifier.md]: As for the deployed contracts, I've realized that this is a "corporate needs you to find the difference between this contract and this contract" situation.... there were no changes to the verifier between 0.9 and 0.10. (Evidence `git diff v0.9.0..v0.10.0 --stat -- contracts`)

File: website/api/blockchain-integration/contracts/verifier.md (hunk 3 above)
Comment (nategraf) [reply website/api/blockchain-integration/contracts/verifier.md]: My intention is to have the "contract" column link to the source code, and the "version" column link to the release.
I've updated the link tah in f2d4a006571565d30422ab77a0cdec3a21c85914

File: website/api/blockchain-integration/contracts/verifier.md (hunk 1 above)
Comment (nategraf) [reply website/api/blockchain-integration/contracts/verifier.md]: Yes! I noticed a number of other places this was broken as well
015ff1d3969a74cba82372679e0a30b2e157568e

File: website/api/blockchain-integration/contracts/verifier.md (hunk 4 above)
Comment (nategraf) [reply website/api/blockchain-integration/contracts/verifier.md]: Best way for someone to check this is to look at the verified source code for the contract on Etherscan, pull the control root out of there, and verify it by recomputing it from the `risc0` repo. We should indeed include instructions on how to do this, probably in a comment above the control root field in the contract source, and potentially elsewhere.

**Pull request merged**